src/schema/schema.graphql

```diff
@@ -290,6 +290,10 @@ type ScratchCodeType {
 	_content: String
 }
 
+type ScratchCodes {
+	scratchCodes: ScratchCode
```

Comment:
shouldn't this be an array? [ScratchCode]

Reply:
It looks like the API doesn't return an Array, https://files.zimbra.com/docs/soap_api/8.8.8/api-reference/zimbraAccount/GenerateScratchCodes.html

Reply:
Yeah, weird implementation of apis :)